content/docs/guides/multi-target-exporter.md

```diff
@@ -0,0 +1,492 @@
+
+## Configuring modules
+
+The modules are predefined in a file inside the docker container called `config.yml` which is a copy of [blackbox.yml](https://github.com/prometheus/blackbox_exporter/blob/master/blackbox.yml) in the github repo.
+
+We will copy this file, [adapt](https://github.com/prometheus/blackbox_exporter/blob/master/CONFIGURATION.md) it to our own needs and tell the exporter to use our config file instead of the one included in the container.  
+
+First download the file using curl or your browser:
+
+```bash
+curl -o blackbox.yml https://raw.githubusercontent.com/prometheus/blackbox_exporter/master/blackbox.yml
+```
+
+Opened in an editor the first few lines look like this:
+
+```yaml
+modules:
+  http_2xx:
+    prober: http
+    http:
+  http_post_2xx:
+    prober: http
+    http:
+      method: POST
+```
+
+[Yaml](https://en.wikipedia.org/wiki/YAML) uses whitespace indentation to express hierarchy, so you can recognise that two `modules` named `http_2xx` and `http_post_2xx` are defined, and that they both have a prober `http` and for one the method value is specifically set to `POST`.  
+We will now change the module `http_2xx` by setting the `preferred_ip_protocol` of the prober `http` explicitly to the string `ip4`.
+
+```yaml
+modules:
+  http_2xx:
+    prober: http
+    http:
+      preferred_ip_protocol: "ip4"
+  http_post_2xx:
+    prober: http
+    http:
+      method: POST
+```
+
+If you want to know more about the available probers and options check out the [documentation](https://github.com/prometheus/blackbox_exporter/blob/master/CONFIGURATION.md).
+
+Now we need to tell the blackbox exporter to use our freshly changed file. We can do that with the flag `--config.file="blackbox.yml"`. But because we are using Docker, we first must make this file [available](https://docs.docker.com/storage/bind-mounts/) inside the container using the `--mount` command.  
+
+NOTE: If you are using macOS you first need to allow the Docker daemon to access the directory in which your `blackbox.yml` is. You can do that by clicking on the little Docker whale in menu bar and then on `Preferences`->`File Sharing`->`+`. Afterwards press `Apply & Restart`.
+
+First you stop the old container by changing into its terminal and press `ctrl+c`.
+Make sure you are in the directory containing your `blackbox.yml`.
+Then you run this command. It is long, but we will explain it:
+
+<a name="run-exporter"></a>
+
+```bash
+docker \
+  run -p 9115:9115 \
+  --mount type=bind,source="$(pwd)"/blackbox.yml,target=/blackbox.yml,readonly \
+  prom/blackbox-exporter \
+  --config.file="/blackbox.yml"
+```
+
+With this command, you told `docker` to:
+
+1. `run` a container with the port `9115` outside the container mapped to the port `9115` inside of the container.
+1. `mount` from your current directory (`$(pwd)` stands for *p*rint *w*orking *d*irectory) the file `blackbox.yml` into `/blackbox.yml` in `readonly` mode.
```

Comment:
Does it? It looks fine in the GitHub file preview.
But my heart is not attached to the emphasis of the initials.
If you think it is better, I can take them away.

Reply:
In the diff view everything afterwards is italic. To me that shows that it's at least ambiguous, Markdown being the underspecified mess that it is. I'd say remove it to avoid issues

Reply:
Well, sad, but okay. I’ll remove the *s.